Documentation/gettingstarted/cni-chaining-azure-cni.rst

```diff
@@ -90,11 +90,14 @@ Deploy Cilium release via Helm:
      --set nodeinit.enabled=true \\
      --set cni.configMap=cni-configuration \\
      --set tunnel=disabled \\
-     --set enableIPv4Masquerade=false
+     --set enableIPv4Masquerade=false \\
+     --set endpointRoutes.enabled=true \\
 
 This will create both the main cilium daemonset, as well as the cilium-node-init daemonset, which handles tasks like mounting the eBPF filesystem and updating the
 existing Azure CNI plugin to run in 'transparent' mode.
 
+``endpointRoutes.enabled`` enables direct routing to the Azure CNI veth pairs without requiring to route via the ``cilium_host`` interface.
```

Comment:
nit: I don't think users will know or necessarily care about this detail - usually we would keep this information either in dedicated developer documentation or just in the commit message.
```suggestion
```